.env.dev

```diff
@@ -0,0 +1,9 @@
+DATA_FOLDER=data-dev
+
+FUSEKI_PORT=3030
+ENDPOINT=http://fuseki:3030
```

Comment:
theoretically, we shall have an unlimited number of ENDPOINTS. It is one of the parameters in the loading the diff. 
Practically it will be a single triple store service, but not sure that is controlled externally.